### DISCUSSION TITLE
Is there an Accessibility Conformance report that lists which accessibility criteria is supported by the Material for MkDocs theme?

### DISCUSSION THREAD

**Discussion by syeda-git** [squidfunk/mkdocs-material#5851] [Q&A] [answered]
VPAT - https://www.section508.gov/sell/vpat/
Sample report - https://www.oxygenxml.com/doc/versions/25.1/ug-webhelp-responsive/topics/whr-vpat.html#whr-vpat__section_sbr_13d_fhb

**Comment by squidfunk** [comment, marked as answer]
No, currently not. We're quite busy pushing the project forward and are happy to improve accessibility as we go. If somebody is up to it to analyze the project and provide such a report, we're happy to promote it and add it to the docs.

**Comment by colleenumn** [comment]
Since the original post in August 2023, a new ADA final rule published in April 2024 now requires WCAG 2.1 Level AA compliance for all digital content provided by state and local governments, including public universities serving 50,000+ residents, by April 24, 2026. ([See ADA.gov Fact Sheet](https://www.ada.gov/resources/2024-03-08-web-rule/))

Given this mandate, would the maintainers consider prioritizing accessibility updates and publishing a compliance roadmap for Material for MkDocs? Doing so would:

- Support legal compliance for public institutions
- Improve access for all users
- Increase adoption across government and higher ed sectors

Happy to contribute testing or documentation efforts if helpful. Thanks for all your work!

**Reply by alexvoss** [reply]
Dear Colleen, 

since Martin's reply to the original question a lot has happened and we are working an [overhaul of Material for MkDocs] that will open up avenues to achieve more comprehensive support for accessibility requirements in the future.

As Martin said in his reply to the original question, he is open to efforts to create an evaluation of Material. We are aware of the increasing importance for compliance with accessibility regulation but as a relatively small open source project, we do not have the resources to conduct a formal evaluation ourselves. We are already involved in a huge undertaking at the moment.

I would be happy to liaise with you if you saw an avenue to source the necessary expertise and resource to conduct an evaluation. Please feel free to reach out to me by email if you would like to talk (see my profile).

Best wishes,

Alex

[overhaul of Material for MkDocs]: https://squidfunk.github.io/mkdocs-material/blog/2024/08/19/how-were-transforming-material-for-mkdocs/

**Comment by dirkpetersen** [comment]
@alexvoss and @colleenumn. I am not sure if this is been currently worked on but i sent a pull request that may at least highlight some issues. https://accessibe.com and https://www.accessibilitychecker.org check a broader spectrum but our immediate issue is  WCAG 21 https://www.w3.org/TR/WCAG21 . If this is not fixed by then we will have to take down our mkdocs-material websites.   I submitted a autogenerated pull request but of course that will still time for review and testing. Happy to discuss more how to tackle this issue if you are interested. 

https://github.com/squidfunk/mkdocs-material/pull/8495

**Reply by alexvoss** [reply]
As mentioned in https://github.com/squidfunk/mkdocs-material/discussions/8461, we are currently working on foundational issues and are looking to bring this work to a close in the coming weeks. It does not make sense for us to work on issues such as WCAG compliance until this is done. Some of the issues that accessibility checkers are picking up will likely be addressed already while others may need additional work.

We are happy to discuss WCAG and the requirements of US legislation on public institutions and work towards a solution for users like yourselves. I also note that the original question was about a compliance report and would like to learn more about this. Please reach out by email, see [my GitHub profile](https://github.com/alexvoss).

Regarding the PR: we cannot merge AI-generated code and some of what Claude did simply duplicates what already exists in Material for MkDocs and just needs to be activated through configuration (breadcrumb navigation, currently in Insiders) or can be achieved through customization (a high-contrast color scheme). Before you take any actions for your projects, I would urge you to contact me. (@squidfunk for visibility)

**Reply by dirkpetersen** [reply]
Thanks much Alex, 

just want to make sure you read my comment in the PR:

<img width="887" height="61" alt="image" src="https://github.com/user-attachments/assets/99e301f2-8cc5-4c92-b853-3cb9c22064fe" />
 
 the intent was to create something as a basis to develop ADA requirements not to merge code.
 
 Let me get ask here who wants to join and then reach out via email.
